modules/microshift-oc-adm-by-example-content.adoc

```diff
@@ -142,52 +56,23 @@ Display information about a release
 ----
   # Show information about the cluster's current release
   oc adm release info
-
+  
   # Show the source code that comprises a release
   oc adm release info 4.11.2 --commit-urls
-
+  
   # Show the source code difference between two releases
   oc adm release info 4.11.0 4.11.2 --commits
-
+  
   # Show where the images referenced by the release are located
   oc adm release info quay.io/openshift-release-dev/ocp-release:4.11.2 --pullspecs
-
+  
   # Show information about linux/s390x image
-  # Note: Wildcard filter is not supported. Pass a single os/arch to extract
+  # Note: Wildcard filter is not supported; pass a single os/arch to extract
   oc adm release info quay.io/openshift-release-dev/ocp-release:4.11.2 --filter-by-os=linux/s390x
 ----
 
 
 
-== oc adm release mirror
-Mirror a release to a different image registry location
-
-.Example usage
-[source,bash,options="nowrap"]
-----
-  # Perform a dry run showing what would be mirrored, including the mirror objects
-  oc adm release mirror 4.11.0 --to myregistry.local/openshift/release \
-  --release-image-signature-to-dir /tmp/releases --dry-run
-
-  # Mirror a release into the current directory
-  oc adm release mirror 4.11.0 --to file://openshift/release \
-  --release-image-signature-to-dir /tmp/releases
-
-  # Mirror a release to another directory in the default location
-  oc adm release mirror 4.11.0 --to-dir /tmp/releases
-
-  # Upload a release from the current directory to another server
-  oc adm release mirror --from file://openshift/release --to myregistry.com/openshift/release \
-  --release-image-signature-to-dir /tmp/releases
-
-  # Mirror the 4.11.0 release to repository registry.example.com and apply signatures to connected cluster
-  oc adm release mirror --from=quay.io/openshift-release-dev/ocp-release:4.11.0-x86_64 \
-  --to=registry.example.com/your/repository --apply-release-image-signature
-----
-
-
-//== oc adm release new
-
 == oc adm taint
 Update the taints on one or more nodes
```

Comment:
We only have one node, so I wouldn't think this should be here, or is should just not say "or more".